lib/src/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, OnInit, Input, OnDestroy } from '@angular/core';
+import { Router, ActivatedRoute, NavigationEnd, Params, PRIMARY_OUTLET } from '@angular/router';
+import 'rxjs/add/operator/filter';
+
+interface IBreadcrumb {
+  label: string;
+  params?: Params;
+  url: string;
+}
+
+@Component({
+    selector: 'hc-breadcrumbs',
+    templateUrl: './breadcrumbs.component.html',
+    styleUrls: ['./breadcrumbs.component.scss']
+})
+export class BreadcrumbsComponent implements OnInit, OnDestroy {
+
+    public breadcrumbs: IBreadcrumb[];
+    public routerSubscription: any;
```

Comment:
I think the appropriate type for routerSubscription would be `Observable<NavigationEnd>`

Reply:
Thanks for the help getting this one resolved @corytak.  Updated in the most recent commit.